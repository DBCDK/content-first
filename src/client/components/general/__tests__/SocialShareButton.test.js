import React from 'react';
import renderer from 'react-test-renderer';

import SocialShareButton from '../SocialShareButton.component';

describe('SocialShareButton', () => {
  it('renders initial component', () => {
    const tree = renderer
      .create(
        <SocialShareButton
          href={'https://content-first.demo.dbc.dk/lister/1234'}
          hex={'#3b5998'}
          size={40}
          shape="round"
          hoverTitle="Del pÃ¥ facebook"
          stamp="123456789"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
