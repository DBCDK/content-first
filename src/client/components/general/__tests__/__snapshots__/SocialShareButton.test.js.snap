// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialShareButton renders initial component 1`] = `
<a
  href="https://laesekompas.dk/lister/1234"
  onClick={null}
  target="_blank"
>
  <button
    className="ssb undefined"
    style={
      Object {
        "backgroundColor": "#3b5998",
        "borderRadius": "50%",
        "height": "40px",
        "paddingLeft": "0px",
        "paddingRight": "0px",
        "width": "40px",
      }
    }
    title="Del pÃ¥ facebook"
  >
    <span
      style={
        Object {
          "marginRight": "0px",
        }
      }
    >
      
    </span>
    
    <i
      className="material-icons"
    />
  </button>
</a>
`;
